bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_run_card.dat

```diff
@@ -58,8 +58,9 @@
 #***********************************************************************
 # PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
 #***********************************************************************
-  lhapdf	=  pdlabel    ! PDF set
-  292200	=  lhaid      ! if pdlabel=lhapdf, this is the lhapdf number
+'lhapdf'    = pdlabel     ! PDF set
+ $DEFAULT_PDF_SETS = lhaid
+ $DEFAULT_PDF_MEMBERS  = reweight_PDF 
```

Comment:
in which case you are using default pdf and lux-qed ?

Reply:
@agrohsje Hey Alexander, let me comment for this question.

The analyser uses photon as the initial state particle for the VBF process (cards that are labeled with "VBF" in the middle) => "generate p a > XXXX"
And because the nominal default PDF set (actually most of the PDFs in the metadata/pdflists.dat) doesn't have photon content, it crashes.

I've asked analysers to specify the 325100 (luxqed) in the run card for the VBF processes instead of modifying the metadata file since there is actually not many possible variations for luxqed PDF (and the analysers won't really consider other luxqed PDF variations other than 325100).

Reply:
But yes a good point, I might have forgotten to comment to the requestors about the systematic PDFs in the runcmsgrid.sh files. I'll ask them through Skype and converge here later.